elasticsearch-rails/lib/rails/templates/seeds.rb

```diff
@@ -53,4 +53,5 @@
 
 # Remove any jobs from the "elasticsearch" Sidekiq queue
 #
+require 'sidekiq/api'
```

Comment:
I got

    uninitialized constant Sidekiq::Queue (NameError)

Where else are you requiring `sidekiq/api`? To me the error and the fix seemed perfectly normal.